.github/problem-matchers/dtc.json

```diff
@@ -0,0 +1,19 @@
+{
+    "problemMatcher": [
+        {
+            "__comment_owner": "match dtc parsing errors",
+            "owner": "dtc",
+            "pattern": [
+                {
+                    "__comment_regexp1": "Error: arch/arm64/boot/dts/qcom/sc7280-idp.dts:27.1-9 Label or path apps_rs not found",
+                    "regexp": "^(Error):\\s+([^:]+):([0-9]+)\\.([0-9]+)(?:[-0-9\\.]+)?\s+(.*)$",
```

Comment:
There's a bare "\s" that should be "\\s". I'm impressed that the github UI showed this -- I would have totally missed this. :)

Reply:
That is certainly impressive! I thought I escaped them all but it does not look like it... Should be fixed now!